### ISSUE TITLE
Plastic Android

### ISSUE THREAD

**Issue by ghost** [studio-minus/ppg-bugs-and-requests#2273]
**Feature description**
You know the regular clunky androids made from steel? Well, this one is made out of brittle plastic. When hit hard enough by a blunt object, it will be smashed, and plastic shards will fly out as particles. It's also lighter and not magnetic

**Feature purpose**
More entity variety, and a smashable plastic effect would be very fun.

**Additional context**
none

**Comment by TeslaCoil9001** [comment]
They could also make a smashable plastic material

**Comment by Drogo-man** [comment]
Why

**Comment by bibbybabby** [comment]
> Why

Check the feature purpose

**Comment by Drogo-man** [comment]
It still makes ZERO sense to me.

**Comment by DindinYT37** [comment]
Sounds like a nice addition, though it kinda sounds like per-pixel-destruction.

**Comment by bibbybabby** [comment]
> Sounds like a nice addition, though it kinda sounds like per-pixel-destruction.

It's not per pixel destruction

**Comment by DindinYT37** [comment]
> > Sounds like a nice addition, though it kinda sounds like per-pixel-destruction.
> 
> It's not per pixel destruction

Yeah, I figured. My point was that it sounds similar to it, which in hindsight, doesn't make that much sense.

**Comment by bibbybabby** [comment]
the procedural gibs is _exactly_ the thing i meant
